filesystem/FileMigrationHelper.php

```diff
@@ -35,7 +35,7 @@ public function run($base = null) {
 		// Loop over all files
 		$count = 0;
 		$originalState = \Versioned::get_reading_mode();
```

Comment:
Yeah that's ok. It's more because at the time of writing I was namespacing everything, but not in this specific change. I'll revert that.